DDCore/include/DDSegmentation/Segmentation.h

```diff
@@ -127,6 +127,14 @@ namespace dd4hep {
           \return vector<double> in natural order of dimensions, e.g., dx/dy/dz, or dr/r*dPhi
       */
       virtual std::vector<double> cellDimensions(const CellID& cellID) const;
+      /// Return true if this segmentation can have cells that span multiple
+      /// volumes.  That is, points from multiple distinct volumes may
+      /// be assigned to the same cell.
+      /// In that case, a working volumeID() implementation is required.
+      virtual bool cellsSpanVolumes() const
```

Comment:
Thanks! That is a better name, and also a docstring that doesn't just use self reference to explain itself :)